eisbuk-admin/src/containers/SlotsPageContainer.tsx

```diff
@@ -55,25 +56,7 @@ const SlotsPageContainer: React.FC<Props> = ({ slots, children, ...props }) => {
 
   const dispatch = useDispatch();
 
-  /**
-   * generates dates to display based on the view
-   * 7 days of the week in case of ice slots
-   * 4-5 Mondays, Tuesdays, etc.of the month in case of off-ice
-   * @param str sview passed from props
-   * @returns array of dates to display
-   */
-  const datesToDisplay =
-    props.view === "slots"
-      ? Array(7)
-          .fill(null)
-          .map((_, i) => currentDate.plus({ days: i }).toISODate())
-      : Array(5)
-          .fill(null)
-          .map((_, i) => currentDate.plus({ week: i }).toISODate())
-          .filter((date) =>
-            DateTime.fromISO(date).hasSame(currentDate, "month")
-          );
-
```

Comment:
Good job on elaborating through JSDoc. No objections there, I've only updated to start with Capital letter and wrote the possible scenarios as bullet points. The mistake here, however, is that it's really uncommon to use JSDoc on variables rather than functions. 
This is ok because it makes more sence to create a function here. Check below please.